src/pymmcore_widgets/_property_browser.py

```diff
@@ -29,6 +30,8 @@ def __init__(
 
         self._prop_table = DevicePropertyTable(mmcore=mmcore)
         self._device_filters = DeviceTypeFilters()
+        with signals_blocked(self._device_filters._read_only_checkbox):
+            self._device_filters._read_only_checkbox.setChecked(True)
```

Comment:
this should just be `self._read_only_checkbox.setChecked(True)` around line 55 in the `DeviceTypeFilters` widget